src/filters.hpp

```diff
@@ -18,14 +18,15 @@ class Filters : public Nan::ObjectWrap {
     using filter_values_type = std::tuple<filter_value_type, filter_properties_type, zoom_type, zoom_type>;
     using filters_type = std::map<filter_key_type, filter_values_type>;
 
+    // ctor
+    static Napi::FunctionReference constructor;
     // initializer
-    static void Initialize(v8::Local<v8::Object> target);
+    static Napi::Object Initialize(Napi::Env env, Napi::Object exports);
+    explicit Filters(Napi::CallbackInfo const& info);
 
-    // method required for the constructor
-    static NAN_METHOD(New); // Filters instance is stored here
-    static NAN_METHOD(layers);
+    Napi::Value layers(Napi::CallbackInfo const& info);
 
-    static auto constructor() -> Nan::Persistent<v8::FunctionTemplate>&;
+    //static auto constructor() -> Napi::FunctionReference&;
```

Comment:
Remove this comment?

Reply:
27be4e200ca7d68bd993caadfa3a052fe5e32f70